src/test/java/com/google/sps/data/DrinkDAOTest.java

```diff
@@ -96,26 +115,31 @@ public void testSearchForDrinkByRating() {
 
   @Test
   public void testSearchForDrink() {
-    Drink a = DrinkDAO.saveDrink(DRINK_A, ratingA, numRatingsA, storeIDA);
-    Drink b = DrinkDAO.saveDrink(DRINK_A, ratingB, numRatingsB, storeIDB);
-    Drink c = DrinkDAO.saveDrink(DRINK_B, ratingC, numRatingsC, storeIDC);
-    Drink d = DrinkDAO.saveDrink(DRINK_B, ratingD, numRatingsD, storeIDD);
+    Drink a = DrinkDAO.saveDrink(DRINK_A, ratingA, storeIDA);
+    Drink b = DrinkDAO.saveDrink(DRINK_A, ratingB, storeIDB);
+    Drink c = DrinkDAO.saveDrink(DRINK_B, ratingC, storeIDC);
+    Drink d = DrinkDAO.saveDrink(DRINK_B, ratingD, storeIDD);
 
     Set<Drink> drinks = Drink.searchForDrink(DRINK_A);
     for (Drink drink : drinks) {
+      Assert.assertTrue(drink.getName().equals(DRINK_A));
       Assert.assertTrue((drink.getStore().equals(storeIDA)) || (drink.getStore().equals(storeIDB)));
```

Comment:
Why is this not deterministic?

Reply:
Since Java set is unordered, I wouldn't know if `DRINK_A` in storeA or `DRINK_A` in storeB would be traversed first

Reply:
A better assertion would be that "drinks" contains _both_ StoreA and StoreB. Currently the test will pass if only one store is returned.